src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
+    static String sectionDivider = "____________________________________________________________";
+    public static void greetUser(){
+        String greeting = "\t" + sectionDivider + "\n"
+                + "\tHello! I'm Duke. \n"
+                + "\tWhat can I do for you? \n"
+                + "\t" + sectionDivider;
+        System.out.println(greeting);
+    }
+    public static void signOff(){
+        String sign_off = "\t" + sectionDivider + "\n"
```

Comment:
Can consider refactoring "sign_off" as signOff since variable names are in camelCase.